src/languages/json-language.js

```diff
@@ -9,6 +9,7 @@
 
 import { parse } from "@humanwhocodes/momoa";
 import { JSONSourceCode } from "./json-source-code.js";
+import { visitorKeys } from "@humanwhocodes/momoa";
```

Comment:
We should update the dependency to `^3.1.1` in package.json.

Reply:
This is so weird. I know I updated this locally and tested, and this is the second time you've pointed out that the version hasn't been updated in `package.json`. 🤔

Reply:
Actually, it is updated in `package.json` but for some reason git isn't included it in the branch.

Reply:
It works if `3.1.1` is locally installed, which in most cases would be the case after doing `npm install` since it's the latest in-range version, but to be sure it should be set in package.json.